server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1769,10 +1895,7 @@ public PoolDTO createPool(@PathParam("owner_key") @Verify(Owner.class) String ow
         pool.setOwner(owner);
 
         // Populate the rest of the pool
-        this.populateEntity(pool, inputPoolDTO);
-
-        // Resolve products
-        //pool = resolverUtil.resolvePool(pool);
+        this.populateEntityOnCreate(pool, inputPoolDTO);
```

Comment:
We shouldn't need separate methods for create/update like this. Do what we can in one, pull the unique bits back into the callers for the rest.